soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2037,6 +2010,8 @@ s8 Player_ItemToItemAction(s32 item) {
         return PLAYER_IA_LAST_USED;
     } else if (item == ITEM_FISHING_POLE) {
         return PLAYER_IA_FISHING_POLE;
+    } else if (item >= ITEM_TUNIC_KOKIRI && item <= ITEM_BOOTS_HOVER) {
```

Comment:
I forgot to ask for you to leave a comment here following the `// SoH:` pattern explaining that this was added to support assignable equipment.

Reply:
Done